README.md

```diff
@@ -1,49 +1,38 @@
-businesses who are dependent on the rights granted by the original, Apache 2.0,
-licence.
-
-To install and run this project, please see the [Downloads Page](https://opensearch.org/downloads.html).
-
-## How you can help
-
-We love issues. The best thing you can do for us is to [start opening issues](https://github.com/opensearch-project/OpenSearch/issues/new/choose).
-Just jump right in and don't hesitate to [contribute](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md)
-if anything strikes your fancy. In case you don't know where to start, we've
-marked a few [open issues](https://github.com/opensearch-project/OpenSearch/issues)
-that would be [really helpful](https://github.com/opensearch-project/OpenSearch/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22).
-There's also a [Developer Guide](https://github.com/opensearch-project/OpenSearch/blob/main/DEVELOPER_GUIDE.md)
-to help you on your way.
-
-Additionally, we have a [forum](https://discuss.opendistrocommunity.dev/)
-dedicated to more general questions and discussions concerning the OpenSearch
-project as a whole. It's a space for community feedback and we'd be grateful to
-hear your opinions and suggestions.
-
-## Guiding Principles
-
-Here are the principles we use to guide our development. From [opensearch.org](http://opensearch.org):
-
-> When we (the contributors) are successful, the OpenSearch project will be:
->
->**1. Great software**. If it doesn’t solve your problems, everything else is moot. It’s going to be software you love to use.
->
->**2. Open source like we mean it**. We are invested in this being a successful open source project for the long term. It’s all Apache 2.0. There’s no Contributor License Agreement. Easy.
->
->**3. A level playing field**. We will not tweak the software so that it runs better for any vendor (including AWS) at the expense of others. If this happens, call it out and we will fix it as a community.
->
->**4. Used everywhere**. Our goal is for as many people as possible to use it in their business, their software, and their projects. Use it however you want. Surprise us!
->
->**5. Made with your input**. We will ask for public input on direction, requirements, and implementation for any feature we build.
->
->**6. Open to contributions**. Great open source software is built together, with a diverse community of contributors. If you want to get involved at any level - big, small, or huge - we will find a way to make that happen. We don’t know what that looks like yet, and we look forward to figuring it out together.
->
->**7. Respectful, approachable, and friendly**. This will be a community where you will be heard, accepted, and valued, whether you are a new or experienced user or contributor.
->
->**8. A place to invent**. You will be able to innovate rapidly. This project will have a stable and predictable foundation that is modular, making it easy to extend.
```

Comment:
These are [Project Principles](https://opensearch.org/#principles) on the website.